sdk/eventhubs/microsoft-azure-eventhubs-eph/pom.xml

```diff
@@ -25,6 +25,20 @@
       <id>azure-java-build-docs</id>
       <url>${site.url}/site/${project.artifactId}</url>
     </site>
+    <relocation>
```

Comment:
Doesn't this require a version update too? In this case `3.1.1`?

Reply:
3.1.0 hasn't been released yet. The latest is 3.0.2, i believe the current versions are correct.